src/main/kotlin/javabot/model/Factoid.kt

```diff
@@ -61,6 +61,11 @@ class Factoid(var name: String = "", var value: String = "", var userName: Strin
         if (name.endsWith(" $^")) {
             replaced = urlencode(camelcase(replacedValue))
         }
+
+        if (replaced.isBlank()) {
+            return null;
+        }
```

Comment:
This is icky, but I couldn't think of a better way do this without a large sweeping change